README.md

```diff
@@ -1,5 +1,7 @@
 # PersonalSite
 
+Old project for a personal website. Never finished.
+
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 6.0.8.
```

Comment:
You should probably also remove the rest of the stuff since it's all boilerplate